android/java/org/chromium/chrome/browser/crypto_wallet/fragments/onboarding/OnboardingRestoreWalletFragment.java

```diff
@@ -113,123 +116,107 @@ public void onTextChanged(CharSequence charSequence, int i, int i1, int i2) {
         secureCryptoButton.setOnClickListener(
                 v -> {
                     String passwordInput = mPasswordEdittext.getText().toString();
-
-                    KeyringService keyringService = getKeyringService();
-                    assert keyringService != null;
-                    keyringService.isStrongPassword(
```

Comment:
Don't we need to validate the password on core side? I remember that was introduced because all 3 platforms had different validation paths and we wanted to unify them.

Reply:
Not anymore, the new password policy has been simplified for all platforms now: the only requirement is >= 8 characters.

https://github.com/brave/brave-core/pull/21051/commits